pkg/restore/pod_action.go

```diff
@@ -64,7 +60,7 @@ func (a *podAction) Execute(obj runtime.Unstructured, restore *api.Restore) (run
 		}
 
 		a.logger.WithField("volumeName", name).Debug("Checking volume")
-		if !tokenRegex.MatchString(name) {
+		if !regexp.MustCompile(serviceAccountName + "-token-.*").MatchString(name) {
```

Comment:
Somewhat nit picky, but is there any value in changing the way we compare these?  The regex vs. `strings.HasPrefix`?

Reply:
It would allow us to drop regexp entirely... Not a bad idea I will fold that in